website/content/docs/plugins/index.mdx

```diff
@@ -1,29 +1,21 @@
 ---
 description: |
-  Packer Plugins allow new functionality to be added to Packer without modifying
+  Packer plugins let you add Packer functionality without modifying
   the core source code. Packer plugins are able to add new builders,
   provisioners, hooks, and more.
 page_title: Plugins
 ---
 
 # Packer Plugins
 
-Packer Plugins allow new functionality to be added to Packer without modifying
-the core source code. **Packer plugins** are able to add new components to
-Packer, such as **builders**, **provisioners**, **post-processors**, and **data
-sources**.
+Packer plugins are separate, standalone applications that perform tasks during each build.
 
-This page documents how to install plugins. You can find a list of available
-plugins in the [Plugins section](/plugins).
+During a `packer build`, the process list shows `packer-` prefixed applications. One of those applications is the Packer binary, and the rest are plugins. Packer launches one plugin process for each component in the build.
 
-If you're interested in developing plugins, see the [developing
-plugins](/docs/plugins/creation#developing-plugins) page.
+The set of builders, provisioners, and post-processors bundled with the Packer binary function as plugins that Packer finds and runs automatically. You can also define a list of external plugins in your template for Packer to run and communicate with throughout the build. These external plugins extend Packer functionality without modifying the core source code.
 
-If you're a plugin maintainer interested in supporting HCP Packer, see the [HCP
-support](/docs/plugins/hcp-support) page.
-
-@include "plugins/how-plugins-work.mdx"
-
-@include "plugins/plugin-tiers-and-namespaces.mdx"
-
-@include "plugins/installing-plugins.mdx"
+Refer to the following plugin documentation:
+- [External Plugins](/plugins) to review the documentation for available packer plugins
+- [Installing Plugins](/docs/plugins/install-plugins) to add external plugins to your Packer template and install the binaries
+- [Developing Plugins](/docs/plugins/creation) to get started creating custom external plugins
+- [HCP Support](/docs/plugins/hcp-support) to update a plugin to support HCP Packer
```

Comment:
The HCP Packer support documentation explains the process for updating a custom builder to so that I can publish its artifacts to the HCP Packer registry. Should this sidebar be moved under the Developing Plugins section? 

Maybe even have the documentation linked to from the Customer Builders and Custom Post-Processors pages. What do you think?

Reply:
I like it :) let me push up a few changes for you to look at!